report_templates/registered_users.erb

```diff
@@ -0,0 +1,26 @@
+<%#
+name: List of registered users
```

Comment:
I'd call it just `Registered users` to remain consistent with other reports. Reports are lists usually, so we don't need to repeat that information